lib/core/src/lib/datatable/components/datatable-cell/datatable-cell.component.ts

```diff
@@ -82,21 +80,10 @@ export class DataTableCellComponent implements OnInit, OnDestroy {
 
     protected onDestroy$ = new Subject<boolean>();
 
-    constructor(protected alfrescoApiService: AlfrescoApiService) {}
+    constructor() {}
 
     ngOnInit() {
         this.updateValue();
-        this.alfrescoApiService.nodeUpdated
```

Comment:
what will be the alternative for this? this feature was to update the document list when node is renamed via the metadata card

Reply:
i think we need to trigger events from that component for ask a refresh or we can add this nodeupdate there if is a content level

Reply:
my solution actually refresh the whole datatable we may need a generic approach to be less invasice I am open to suggestions

Reply:
@DenysVuika check now